src/main/java/org/broadinstitute/hellbender/utils/codecs/gencode/GencodeGtfFeature.java

```diff
@@ -763,6 +763,8 @@ public GencodeGtfFeature create(final String[] gtfFields) {
             }
         };
 
+        private static final FeatureType[] valueArray = values();
```

Comment:
Java code conventions are to use all uppercase for static final fields, e..g `VALUES`

Also, I'm not sure why you need to create a static array for this object. If you want to converting from string very fast, you could use a static `Map`.

Reply:
I wanted bi-directional conversion (string <-> enum) without case sensitivity, so this seemed fairly straightforward.  I can refactor to use a `Map`.

I have no problem renaming the field to `VALUEMAP` once the above refactor is complete.